Examples/Tests/ohm_solver/ion_Landau_damping/PICMI_inputs.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python3
+#
+# --- Test script for the kinetic-fluid hybrid model in WarpX wherein ions are
+# --- treated as kinetic particles and electrons as an isothermal, inertialess
+# --- background fluid. The script simulates ion Landau damping as descibed
+# --- in section 4.5 of Munoz et al. (2018).
+
+import argparse
+import os
+import sys
+import time
+
+import dill
+from mpi4py import MPI as mpi
+import numpy as np
+
+from pywarpx import callbacks, fields, picmi
+
+constants = picmi.constants
+
+comm = mpi.COMM_WORLD
+
+simulation = picmi.Simulation(verbose=0)
+# make a shorthand for simulation.extension since we use it a lot
+sim_ext = simulation.extension
+
+
+class IonLandauDamping(object):
+    '''This input is based on the ion Landau damping test as described by
+    Munoz et al. (2018).
+    '''
+    # Applied field parameters
+    B0          = 0.1 # Initial magnetic field strength (T)
+    beta        = 2.0 # Plasma beta, used to calculate temperature
+
+    # Plasma species parameters
+    m_ion      = 100.0 # Ion mass (electron masses)
+    vA_over_c  = 1e-3 # ratio of Alfven speed and the speed of light
+
+    # Spatial domain
+    Nz          = 256 # number of cells in z direction
+    Nx          = 4 # number of cells in x (and y) direction for >1 dimensions
+
+    # Temporal domain (if not run as a CI test)
+    LT          = 40.0 # Simulation temporal length (ion cyclotron periods)
+
+    # Numerical parameters
+    NPPC        = [8192, 4096, 1024] # Seed number of particles per cell
+    DZ          = 1.0 / 6.0 # Cell size (ion skin depths)
+    DT          = 1e-3 # Time step (ion cyclotron periods)
+
+    # density perturbation strength
+    epsilon = 0.03
+
+    # Plasma resistivity - used to dampen the mode excitation
+    eta = 1e-7
```

Comment:
Out of curiosity: is this technically needed here? What happens if we set this to 0.
(Also: do I understand correctly that the theoretical formula that we compare the simulation with assumes 0 resistivity?)

Reply:
It is needed just for numerical stability. Inclusion of the resistivity doesn't shift the dispersion though, it only decreases the intensity.